src/cljs/lumo/classpath.cljs

```diff
@@ -0,0 +1,45 @@
+(ns lumo.classpath
+  (:require [clojure.string :as string]))
+
```

Comment:
Also, I'd like to see a `filenames-in-jar` function a la https://github.com/clojure/java.classpath/blob/master/src/main/clojure/clojure/java/classpath.clj#L50